model/registry/process.yaml

```diff
@@ -165,8 +184,20 @@ groups:
         brief: >
           The exit code of the process.
         examples: [127]
+      - id: uptime
+        type: int
+        stability: experimental
+        brief: >
+          Seconds the process has been up.
+        examples: [2880]
       - id: interactive
         type: boolean
         stability: experimental
         brief: >
           Whether the process is connected to an interactive shell.
+      - id: working_directory
```

Comment:
hum couldn't this be considered sensitive information? 🤔

Reply:
What sort of sensitive information are you thinking of?

It's possible that it contains something that someone considers sensitive, but it's also possible that a few of the other existing fields also have similar sensitive information, like `executable.path`, `user.id`, `command_args`, etc.

Do you think this needs any special call-out on potentially having sensitive information?

Reply:
I see from [here](https://github.com/open-telemetry/semantic-conventions/pull/731#issuecomment-2072875648) there's already sensitive tags, which I didn't know about. Maybe it should be a larger conversation on how to apply them to the process fields, since there are a lot of other fields that could potentially have sensitive data

Reply:
there is already a larger [discussion](https://github.com/open-telemetry/oteps/pull/255/files) on sensitive data
After resolution for this otep we would need to mark probably all sensitive fields in our registry. But for now I don't think we need to change this PR and remove this field. Maybe add a note to it. But I agree with you, this field is as sensitive as many other process fields, for example command args could have some credentials passed over etc